onnx_caffe2/backend.py

```diff
@@ -471,18 +484,30 @@ def prepare(cls, model, device='CPU', **kwargs):
         '''
         super(Caffe2Backend, cls).prepare(model, device, **kwargs)
 
-        init_net, predict_net = cls.onnx_graph_to_caffe2_net(model.graph)
-        predict_net.device_option.CopyFrom(get_device_option(Device(device)))
+        ws = Workspace()
 
+        # Directly load initializer data into blobs in workspace
+        cls._direct_initialize_parameters(model.graph.initializer, ws)
+        # Need to pull this out before we delete model.graph.initializer
         initialized = {init.name for init in model.graph.initializer}
+        # Delete the initializers so they aren't serialized
+        del model.graph.initializer[:]
```

Comment:
add this back to the `model` at the end of prepare, it's better not to mutate the function input.